pysat/utils/registry.py

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
The pysat docs are driven by the docstrings. More background and exposition is preferred.

Reply:
Instantiating a pysat.Instrument object for a particular data set requires a module for each instrument with routines that download, load, and clean the respective data. This module enables pysat to provide the same user experience for external or custom instrument modules, as for those shipped with pysat, by maintaining information about these outside instrument files.

Reply:
Instrument support modules must be registered before use, for example:
blah blah

Reply:
I have improved the documentation and added a reference to https://pysat.readthedocs.io/en/latest/new_instrument.html